### ISSUE TITLE
[FR] Use checksum in the cache

### ISSUE THREAD

**Issue by sk-** [smeijer/unimported#84]
Currently the cache uses only the `mtime` and `size`, which unfortunately does not work on CI environments because git does not restore the `mtime`.

Is for this reason that I propose we use the checksum of the files, which is already supported by the underlying library. See https://www.npmjs.com/package/file-entry-cache#user-content-createfromfilepathtocache-usechecksum. This could be either set permanently or controlled by a flag, similar to what `eslint` does:

```
  --cache-strategy String         Strategy to use for detecting changed files in the cache - either: metadata or content - default:
                                  metadata
```

If instead of making it an option, we just set the value to always use the checksum, then the change would be really straightforward and would just be a matter of adding `, true` to the call 
https://github.com/smeijer/unimported/blob/fa7aa13d647f4526d2bfff4a765e3f659ff459e1/src/cache.ts#L32-L35

I'm asking for this feature because in our case running `unimported` is taking a long time.

Github Actions (cache not working): 56s

Locally without cache:
```
real	2m25.659s
user	2m54.552s
sys	0m21.372s
```

Locally with cache:
```
real	0m9.382s
user	0m10.217s
sys	0m3.792s
```

**Comment by sk-** [comment]
For the time being we are patching `unimported` in our CI with the following command:

```
sed -E -i".bak" "s/default.create\((.*)\);/default.create(\1, true);/" node_modules/unimported/dist/cache.js && npx unimported
```

**Comment by smeijer** [comment]
Thanks. Great suggestion! I've decided to use this as the new default. It will be part of the next release.

**Comment by smeijer** [comment]
Released in v1.20.0
